roles/nfs/node/tasks/install_remote_pkg.yml

```diff
@@ -73,57 +73,57 @@
 - name: install | nfs path
   set_fact:
    scale_nfs_url: 'ganesha_rpms/rhel7/'
-  when: ansible_distribution == 'RedHat' and ansible_distribution_major_version == '7'
+  when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '7'
 
 - name: install | nfs path
   set_fact:
    scale_nfs_url: 'ganesha_rpms/rhel8/'
-  when: ansible_distribution == 'RedHat' and ansible_distribution_major_version == '8'
+  when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '8'
 
 - name: install | nfs path
   set_fact:
    scale_nfs_url: 'ganesha_debs/ubuntu16/'
-  when: ansible_distribution == 'Ubuntu'
+  when: ansible_distribution in scale_ubuntu_distribution
 
 - name: install | nfs path
   set_fact:
    scale_nfs_url: 'ganesha_rpms/sles12/'
-  when: ansible_distribution == 'SLES'
+  when: ansible_distribution in scale_ubuntu_distribution
```

Comment:
Looks like this should be `scale_sles_distribution`

Reply:
God Catch.   should this also be `ansible_distribution_major_version == '12'`